app/views/track.js

```diff
@@ -51,14 +52,81 @@ const TrackView = Mn.LayoutView.extend({
                 this.setClass();
             }
         }, this);
+
+        // Reorder the track on drag and drop
+        this.listenTo(application.channel,'playlist:reorder', (tracks) => {
+            if (tracks.get(this.model.get('_id'))) {
+                this.setClass();
+                this.setOrder();
+            }
+        }, this);
+
+        $(document).mousemove((e) => {
+            if (this.dragDown) {
+                this.drag(e);
+            }
+        });
+        $(document).mouseup((e) => {
+             if (this.dragDown) {
```

Comment:
ProTip: you can write

```js
if (!this.dragDown) return
this.dragDown = false
/* ... */
```

The early return will make code more easy to read and prevent pyramid if tests ;)